### ISSUE TITLE
Building Error

### ISSUE THREAD

**Issue by mudongliang** [josepablocam/phosphor-examples#1]
mvn package building has error! And the same output with run_examples.sh!
![image](https://cloud.githubusercontent.com/assets/6214861/13798454/f5d82890-eaec-11e5-9914-1ae43045a492.png)

**Comment by josepablocam** [comment]
I have sparse access to my laptop over the next couple of days, but will
take a look and post when fixed. Thanks

On Tue, Mar 15, 2016 at 8:32 PM, 慕冬亮 notifications@github.com wrote:

> [image: image]
> https://cloud.githubusercontent.com/assets/6214861/13798454/f5d82890-eaec-11e5-9914-1ae43045a492.png
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> https://github.com/josepablocam/phosphor-examples/issues/1

## 

Jose Pablo Cambronero

**Comment by josepablocam** [comment]
The phosphor jar has to be installed in the local maven repository for this build in phosphor-examples to work. This was not well documented. I have modified the README to make this clearer.

./run_examples.sh won't work until `mvn package` successfully completes (if it cannot find the jar for the project, it runs mvn package, which explains why you saw the same error). This should take care of both issues.

**Comment by josepablocam** [comment]
@mudongliang sorry accidentally closed. Can you run with the updated instructions and verify it works on your machine? (I tried on mine and had no issues)

**Comment by mudongliang** [comment]
![image](https://cloud.githubusercontent.com/assets/6214861/13933029/89ca28b2-ef81-11e5-9468-44e6abf37457.png)

I have create phosphor jar and instrumented jre version.
Then I execute the shell script but error occurs.
Do my commands have problems?

**Comment by josepablocam** [comment]
Did you run `mvn install` in the Phosphor folder as well?

**Comment by mudongliang** [comment]
No, I don't. When I execute this command, it's OK now. Thanks.

**Comment by qorost** [comment]
The present version of phosphor has updated to `0.03`, same error will appear.

Solution is to : replace the `version` in `pom.xml`.

**Comment by josepablocam** [comment]
@qorost If I recall correctly, this project expected you to build phosphor outside (not as part of it), so there may be other things to change besides just the version in the pom. Mind putting in a PR if you're interested?

**Comment by qorost** [comment]
@josepablocam , yes, it is expected to build phosphor outside, but in the phosphor project the version number is `0.0.3`. Do you want me to put a PR with the POM.xml modified?
